togetherly/lib/providers/chore_provider.dart

```diff
@@ -25,19 +26,42 @@ class ChoreProvider with ChangeNotifier {
 
   List<Chore> _allChores = [];
   List<Chore> get allChores => _allChores;
+  List<Chore> get choreListDueToday => _allChores
+      .where((chore) => chore.dueDate == DateHelpers.getDateToday())
+      .toList();
+  List<Chore> get choreListComingSoon =>
+      _allChores.where((chore) => _isChoreDueTomorrow(chore.dueDate)).toList();
+  List<Chore> get choreListOverdue =>
+      _allChores.where((chore) => _isChoreOverdue(chore.dueDate)).toList();
+
+  bool _isChoreDueTomorrow(DateTime dueDate) {
+    final dateToday = DateHelpers.getDateToday();
+    return dueDate.year == dateToday.year &&
+        dueDate.month == dateToday.month &&
+        dueDate.day == dateToday.day + 1;
+  }
+
+  bool _isChoreOverdue(DateTime dueDate) {
+    final dateToday = DateHelpers.getDateToday();
+    return dueDate.year <= dateToday.year &&
+        dueDate.month <= dateToday.month &&
+        dueDate.day < dateToday.day;
```

Comment:
I think `&&` might need to be replaced with `||` in this function.

Reply:
Fixed this logic.